### DISCUSSION TITLE
Enhancement: The idle animations are synced as the old fnf versions (0.2.7.1-)

### DISCUSSION THREAD

**Discussion by Danii2008** [FunkinCrew/Funkin#5098] [Q&A]
## What is your suggestion, and why should it be implemented?

I noticed that the Boyfriend and Dad (Opponent) idle animations are synced every beat and not every 2 beats (that means they will play the idle regardless there's a camera zoom or not). This was changed in 0.2.8 and brought back in 0.3.0.

I don't know if this was intentional or not, i just say this in case that this wasn't the intention.

P.S: Sorry if you didn't understand, my english is not the best. If you have questions about this go ahead and ask me.

**Comment by AnniIsHere** [comment]
Its intentional when the bpm is really low so that the characters aren't static for too long

**Comment by Danii2008** [comment]
> Its intentional when the bpm is really low so that the characters aren't static for too long

They can probably code something like if the bpm is 100 or lower the characters do their idle anims every beat. I tried to do this and it works pretty well.

**Comment by AbnormalPoof** [comment]
what
